packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -633,15 +633,24 @@ class ScrollableBase extends Component {
 		const
 			isRtl = this.uiRef.current.state.rtl,
 			isPreviousScrollButton = direction === 'up' || (isRtl ? direction === 'right' : direction === 'left'),
-			isVerticalScrollBar = direction === 'up' || direction === 'down';
+			isHorizontalDirection = direction === 'left' || direction === 'right',
+			isVerticalDirection = direction === 'up' || direction === 'down';
 
-		this.onScrollbarButtonClick({isPreviousScrollButton, isVerticalScrollBar});
+		this.onScrollbarButtonClick({isPreviousScrollButton, isVerticalDirection});
```

Comment:
We're changing the args to `onScrollbarButtonClick` but that function hasn't been updated to use the new member (which is relevant because we're passing as an obj rather than fn args).